### PR TITLE
Project Hello World

### PR DESCRIPTION
**Pull request by EuroCigan**

### CODE REVIEW

File: src/learning/Main.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package src.learning;
+
+public class Main {
+    public static void main(String[] args) {
+        System.out.println("Hello World");
+    }
```
Comment (Postoronnik) [inline src/learning/Main.java:6]: Your translator is shit

**Review by Postoronnik — CHANGES_REQUESTED** [review]

File: src/learning/Main.java (hunk 1 above)
Comment (EuroCigan) [reply src/learning/Main.java:6]: gsijd nb

**Review by Postoronnik — COMMENTED** [review]
sdafsdf

File: learning-git.iml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
```
Comment (Postoronnik) [inline learning-git.iml:8]: asdasd

**Review by Postoronnik — APPROVED** [review]

**Pull request merged**